### PR TITLE
freerun changes per nerf day

### PR DESCRIPTION
**Pull request by horrible-little-slime**

### CODE REVIEW

File: packages/garbo/src/diet.ts [hunk 1]
```diff
@@ -438,7 +438,7 @@ function menu(): MenuItem<Note>[] {
     new MenuItem($item`frozen banquet`),
     new MenuItem($item`deviled egg`),
     new MenuItem($item`spaghetti breakfast`, { maximum: spaghettiBreakfast }),
-    new MenuItem($item`extra-greasy slider`, { maximum: 1 }),
+    new MenuItem($item`extra-greasy slider`),
```
Comment (Rinn) [inline packages/garbo/src/diet.ts:441]: ???

File: packages/garbo/src/diet.ts [hunk 2]
```diff
@@ -458,7 +458,7 @@ function menu(): MenuItem<Note>[] {
     new MenuItem($item`splendid martini`),
     new MenuItem($item`low tide martini`),
     new MenuItem($item`Eye and a Twist`),
-    new MenuItem($item`jar of fermented pickle juice`, { maximum: 1 }),
+    new MenuItem($item`jar of fermented pickle juice`),
```
Comment (Rinn) [inline packages/garbo/src/diet.ts:461]: ???

**Comment by horrible-little-slime** [comment]
Handled by libram update

File: packages/garbo/src/diet.ts (hunk 1 above)
Comment (horrible-little-slime) [reply packages/garbo/src/diet.ts:441]: Covered by the libram update

Comment (horrible-little-slime) [reply packages/garbo/src/diet.ts:441]: And made doubly redundant now by #1815

**Review by Malibu-Stacey — APPROVED** [review]

**Pull request merged**